lib/fluent/event.rb

```diff
@@ -21,6 +21,10 @@ class EventStream
     include Enumerable
     include MessagePackFactory::Mixin
 
+    def records
```

Comment:
`events` is better for me.

@sonots How about this? This method will be used in flowcounter_simple or similar gems.